### DISCUSSION TITLE
Sorting Alerts

### DISCUSSION THREAD

**Discussion by joeb1kenobe** [Security-Onion-Solutions/securityonion#3107] [Ideas]
I like to sort alerts by event_severity_label. However, when I short using the header, the events get sorted alphabetically. So, I get alerts orders High, Low, Medium. But I would like to see them in High, Medium, Low order. 

Is there a way to change the default sorting in the Alerts page to allow the severity to be sorted in High, Medium, Low order?

Thank you.

**Comment by weslambert** [comment]
It seems to be working okay for me when I click `event.severity_label`, but I may be misinterpreting:

![image](https://user-images.githubusercontent.com/16829864/109854103-a4c56980-7c24-11eb-8adb-305ae73baac3.png)

**Reply by joeb1kenobe** [reply]
Here is what I see on my screen when I sort the same way.

![image](https://user-images.githubusercontent.com/519034/109855110-e571b280-7c25-11eb-91a9-6bcc8a023b43.png)
